app/src/AlgoliasearchZendeskHC.js

```diff
@@ -23,6 +23,7 @@ const optionsStructure = {required: true, type: 'Object', children: {
   }},
   baseUrl: {type: 'string', value: '/hc/'},
   color: {type: 'string', value: '#158EC2'},
+  clickAnalyticsEnabled: {type: 'boolean', value: false},
```

Comment:
Maybe the same way we have `analytics: true`, here `clickAnalytics: true` would work too?

Reply:
You mean re-naming `clickAnalyticsEnabled` to `clickAnalytics` ? I named it differently to not confuse it with the search parameter that has the same name, but yes, I guess there's no problem them having the same name 👍